signingscript/docker.d/init_worker.sh

```diff
@@ -160,9 +160,9 @@ case $ENV in
         test_var_set 'AUTOGRAPH_AUTHENTICODE_SHA2_PASSWORD'
         test_var_set 'AUTOGRAPH_AUTHENTICODE_SHA2_USERNAME'
         test_var_set 'AUTOGRAPH_MAR_PASSWORD'
-        test_var_set 'AUTOGRAPH_MAR_STAGE_PASSWORD'
-        test_var_set 'AUTOGRAPH_MAR_STAGE_USERNAME'
         test_var_set 'AUTOGRAPH_MAR_USERNAME'
+        test_var_set 'AUTOGRAPH_STAGE_MAR_PASSWORD'
+        test_var_set 'AUTOGRAPH_STAGE_MAR_USERNAME'
         test_var_set 'AUTOGRAPH_GPG_PASSWORD'
         test_var_set 'AUTOGRAPH_GPG_USERNAME'
         test_var_set 'GPG_PUBKEY_PATH'
```

Comment:
Might want to take the opportunity to add the xpi signing variables here, that I forgot when adding xpi stage signing to  adhoc-t.